runtime/moonbase/tests/common/mod.rs

```diff
@@ -96,7 +95,6 @@ pub fn run_to_block(n: u32, author: Option<NimbusId>) {
 		Ethereum::on_initialize(System::block_number());
 
 		// Finalize the block
-		Ethereum::on_finalize(System::block_number());
```

Comment:
Why remove this?

Reply:
Because it slows down the test, and is not even used in for the purpose of the test, causing a timeout in the CI. It was also removed for moonbeam and moonriver.